tools/nl/validator/placetest.py

```diff
@@ -34,7 +34,9 @@
                     'Unique name of the test, for continuation, etc.')
 flags.DEFINE_bool('do_places_in', False, 'Generate places in?')
 
-URL = 'https://api.datacommons.org/v1/recognize/places?key=AIzaSyCTI4Xz-UW_G2Q2RfknhcfdAnTHq5X5XuI'
+# Use autopush endpoint for recon to get the latest fixes.
+URL = 'https://autopush.api.datacommons.org/v1/recognize/places?key=AIzaSyBCybF1COkc05kj5n5FHpXOnH3EdGBnUz0'
```

Comment:
Do we want to leave the key in there? Can this be picked from the environment?

Reply:
Done.